Formula/fregerepl.rb

```diff
@@ -0,0 +1,29 @@
+class Fregerepl < Formula
```

Comment:
I honestly didn't know what the naming conventions should have been as this is my first ever contribution to this project! I have changed everything per your suggestions. ^_^